test/instrumentation/rails_action_view_test.rb

```diff
@@ -18,7 +18,7 @@ def app
   def setup
     clear_all!
     @framework_version = Gem::Specification.find_by_name('rails').version
-    @supported_framework_version = @framework_version < Gem::Version.new('6.1')
+    @supported_framework_version = @framework_version < Gem::Version.new('7.2')
```

Comment:
I think now all these from line `20` to `31` should be removed, and also the calls to these lambdas.